liferay-workspace/build.gradle

```diff
@@ -0,0 +1,129 @@
+FileTree liferayGradleOutputFiles = fileTree(dir: '../liferay-gradle/', include: '**/libs/*.jar')
+
+buildscript {
+        repositories {
+                jcenter()
+        }
+
+        dependencies {
+        		classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:3.2.0'
+        		classpath 'com.jcabi:jcabi-manifests:1.1'
+                classpath 'com.squareup.okhttp3:okhttp:3.3.1'
+                classpath 'de.undercouch:gradle-download-task:2.1.0'
+        }
+}
+
+apply plugin: 'biz.aQute.bnd.builder'
+apply plugin: 'de.undercouch.download'
+
+import com.jcabi.manifests.Manifests
+import java.util.jar.JarInputStream;
+import de.undercouch.gradle.tasks.download.Download;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+def bladeclijar = 'com.liferay.blade.cli_1.0.1.201606141418.jar'
+
+task downloadBladeCLI(type: Download) {
+    dest buildDir
+    overwrite false
+    src "https://liferay-test-01.ci.cloudbees.com/job/liferay-blade-cli/21/artifact/build/generated/p2/plugins/${bladeclijar}"
```

Comment:
You can use the new official release: http://releases.liferay.com/tools/blade-cli/1.0.1.201606142156/plugins/com.liferay.blade.cli_1.0.1.201606142156.jar